### PR TITLE
xorg/system: install libxcb-util0-dev on ubuntu AND debian, libxcb-ut…

### PR DESCRIPTION
**Pull request by ledocc**
…il-dev for other apt based distro

Specify library name and version:  **xorg/system**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by Croydon — APPROVED** [review]
The current dev version of Debian has `libxcb-util-dev` too, this should be included in the condition

https://pkgs.org/search/?q=libxcb-util-dev

**Review by ericLemanissier — DISMISSED** [review]

**Review by SSE4 — APPROVED** [review]

**Comment by Croydon** [comment]
Lol, I have no clue why I approved, I wanted to post this as a comment

**Review by Croydon — CHANGES_REQUESTED** [review]
The current dev version of Debian has libxcb-util-dev too, this should be included in the condition

https://pkgs.org/search/?q=libxcb-util-dev

File: recipes/xorg/all/conanfile.py [hunk 1]
```diff
@@ -42,7 +42,11 @@ def system_requirements(self):
             if tools.os_info.with_apt:
                 packages = ["xorg-dev", "libx11-xcb-dev", "libxcb-render0-dev", "libxcb-render-util0-dev", "libxcb-xkb-dev",
                             "libxcb-icccm4-dev", "libxcb-image0-dev", "libxcb-keysyms1-dev", "libxcb-randr0-dev", "libxcb-shape0-dev",
-                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data", "libxcb-util0-dev" if tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version < "15" else "libxcb-util-dev"]
+                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data"]
+                if (tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version < "15") or tools.os_info.linux_distro == "debian" :
```
Comment (Croydon) [inline recipes/xorg/all/conanfile.py:46]:
```suggestion
                if (tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version < "15") or (tools.os_info.linux_distro == "debian" and tools.os_info.os_version < "12"):
```

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by Croydon — APPROVED** [review]

**Pull request merged**